chatt-app/ChatServer.java

```diff
@@ -1,93 +0,0 @@
-import java.io.*;
-import java.net.*;
-import java.util.StringTokenizer;
-import java.util.Vector;
-
-
-public class ChatServer {
-	static Vector ClientSockets;
-	static Vector LoginNames;
-	
-	ChatServer() throws IOException{
-		ServerSocket server = new ServerSocket(5217);
-		ClientSockets = new Vector();
-		LoginNames = new Vector();
-		
-		while(true){
-			Socket client = server.accept();
-			AcceptClient acceptClient = new AcceptClient(client);

```

Comment:
could potentially put client socket inside constructor for AcceptClient as well. Only if acceptClient would be used.